<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="key_tab">
       <attribute name="title">
        <string>Ключ</string>
       </attribute>
       <widget class="QPushButton" name="key_generate_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>261</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Сгенерировать ключ шифрования</string>
        </property>
       </widget>
       <widget class="QPushButton" name="key_input_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>371</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ввести существующий ключ шифрования</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="exist_key_editor">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="generated_key_editor">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>230</y>
          <width>411</width>
          <height>71</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Тут появится сгенерированный ключ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="encrypt_tab">
       <attribute name="title">
        <string>Зашифровать</string>
       </attribute>
       <widget class="QPushButton" name="encrypt_button">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>290</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Зашифровать</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="encrypt_input_editor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>351</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Входные данные</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="encrypt_output_editor">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>391</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Выходные данные</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="decrypt_tab">
       <attribute name="title">
        <string>Расшифровать</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Входные данные</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="decrypt_input_editor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>351</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="decrypt_output_editor">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>391</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Выходные данные</string>
        </property>
       </widget>
       <widget class="QPushButton" name="decrypt_button">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>290</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Расшифровать</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
